sensors/nxt_i2c_sensor_defs.c

```diff
@@ -984,11 +997,13 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 				/**
 				 * @description: Angle
 				 * @units_description: degrees
-				 * @value0: Angle (0 to 180)
+				 * @value0: Angle (0 to 359)
 				 */
 				.name = "ANGLE",
-				.raw_max = 180,
-				.si_max = 180,
+				.data_sets = 2,
```

Comment:
This only returns one value here correct? `data_sets` should be 1, not 2, otherwise `value1` will return a meaningless value. Instead, set `data_type` to get the i2c to read two bytes. `'LEGO_SENSOR_DATA_U16` would be appropriate, I think.